### PR TITLE
Improve error msg when using libyui REST in product version <15SP3

### PR DESCRIPTION
**Pull request by skliu**
Improve libyui REST error message.

- Related ticket: https://progress.opensuse.org/issues/112832
- Needles: -
- Verification run:
SLE15SP2: http://d218.qam.suse.de/tests/55
SLE15SP4: http://d218.qam.suse.de/tests/56
openSUSE15.3: http://d218.qam.suse.de/tests/57
openSUSE15.4: http://d218.qam.suse.de/tests/58

### CODE REVIEW

**Comment by asdil12** [comment]
Could you update the commit message so that it fits in one line.
eg:
```
Improve error msg when using libyui REST in product version <15SP3
```

**Review by asdil12 — APPROVED** [review]

File: schedule/yast/maintenance/yast_libyui.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+vars:
+  DESKTOP: textmode
+  YUI_REST_API: 1
+  YUI_HTTP_PORT: 39125
+  YUI_HTTP_REMOTE: 1
+  YUI_REUSE_PORT: 1
+  
+schedule:
+  - installation/bootloader
+  - installation/setup_libyui
```
Comment (ge0r) [inline schedule/yast/maintenance/yast_libyui.yaml:11]: Why do we need this yaml file?

Comment (jknphy) [reply schedule/yast/maintenance/yast_libyui.yaml:11]: this was just for testing, please, remove

**Comment by jknphy** [comment]
@skliu please rebase your code.

File: schedule/yast/maintenance/yast_libyui.yaml (hunk 1 above)
Comment (skliu) [reply schedule/yast/maintenance/yast_libyui.yaml:11]: got it.

**Pull request merged**